package.json

```diff
@@ -6,7 +6,12 @@
   "directories": {
     "test": "test"
   },
+  "watch": {
```

Comment:
could keep it and have the same thing in the client, thought less keys would be cleaner =P